Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm

```diff
@@ -2240,9 +2245,19 @@ - (void)didEndWritingToolsSession:(WTSession *)session accepted:(BOOL)accepted
     _activeWritingToolsSession = nil;
     [_writingToolsTextSuggestions removeAllObjects];
 
-    _page->setWritingToolsActive(false);
```

Comment:
Why is this no longer being called if `session.type != WTSessionTypeProofreading`?